medicines/doc-index-updater/src/storage_client/azure_blob_client.rs

```diff
@@ -0,0 +1,129 @@
+use super::{
+    models::{StorageClientError, StorageFile},
+    GetBlob, StorageClient,
+};
+use crate::create_manager::hash;
+use async_trait::async_trait;
+use azure_sdk_core::{
+    BlobNameSupport, BodySupport, ContainerNameSupport, ContentMD5Support, ContentTypeSupport,
+    MetadataSupport,
+};
+use azure_sdk_storage_blob::Blob;
+use azure_sdk_storage_core::client::Client;
+use std::collections::HashMap;
+
+pub struct AzureBlobStorage {
+    pub container_name: String,
+    prefix: String,
+    storage_account: String,
+    master_key: String,
+}
+
+impl Default for AzureBlobStorage {
+    fn default() -> Self {
+        Self::temporary()
+    }
+}
+
+impl AzureBlobStorage {
+    pub fn temporary() -> Self {
+        let container_name =
+            std::env::var("STORAGE_CONTAINER").expect("Set env variable STORAGE_CONTAINER first!");
```

Comment:
decouple from `permanent()` below by switching the `STORAGE_CONTAINER` for `TEMPORARY_STORAGE_CONTAINER`, which we can keep to the same value for now, but we should change with a new container created by terraform asap